### ISSUE TITLE
How to add custom ops to Tensorflow Lite Swift/ObjC ? 

### ISSUE THREAD

**Issue by scm-ns** [tensorflow/tensorflow#34183]
I can't find any straight forwards docs on how to add a custom op to tensorflow lite, swift/objc. I see the docs for adding to the c++ lib, but it seems unclear how to add them to the swift pods? 

Does anyone have any pointers?

**Comment by gadagashwini** [comment]
@scm-ns, Please take a look at this [link](https://www.tensorflow.org/lite/guide/ops_custom). Thanks!

**Comment by scm-ns** [comment]
@gadagashwini It tells me how to add it to C++. Not how to run it within an iOS app. 

If you look at this line `builtins.AddCustom("Sin", Register_SIN());`. I am not sure how to do this within an iOS app.

**Comment by tpeet** [comment]
Hi, 

I trained a model and converted it to TFLite with select ops model with TF2.0. In Android it works well, after including tensorflow-lite-select-tf-ops.aar from repository, but as I understand, for iOS the lite-with-select-ops CocoaPod is not yet published and we should compile it ourselves. The problem is that in TF 2.0 [documentation ](https://www.tensorflow.org/lite/guide/ops_select) it suggests to run `tensorflow/contrib/makefile/build_all_ios_with_tflite.sh`, which was removed in the new version of Tensorflow. Any suggestions how to compile it? When could we expect CocoaPod for lite-with-select-ops?

Thanks!
